### PR TITLE
Add SimpleNote version 1.5.0 (#1879)

### LINKED ISSUE #1879: [Request] Add SimpleNote
SimpleNote is a simple note taker, based on Electron.

https://simplenote.com

### PR DESCRIPTION
**Pull request by eliezio**
Close #1803 and Close #1879

### CODE REVIEW

File: bucket/simplenote.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    },
+    "post_install": "Remove-Item \"$dir\\app-64.7z\", \"$dir\\app-32.7z\"",
+    "bin": "SimpleNote.exe",
+    "shortcuts": [
+        [
+            "SimpleNote.exe",
+            "SimpleNote"
+        ]
+    ],
+    "checkver": {
```
Comment (Ash258) [inline bucket/simplenote.json]: They use normal number releaes. No need to make it this hard and strict it to stable releases

```suggestion
    "checkver": {
        "github": "https://github.com/Automattic/simplenote-electron"
    }
```

**Comment by eliezio** [comment]
It seems that one of the AppVeyor failed due to an internal error. I'm going to cancel this pull request and re-submit another one.

**Comment by Ash258** [comment]
No need to open new PR. First stage passed, only powershell core have some troubles, but it's appveyors fault.

**Comment by eliezio** [comment]
Sorry but I had already canceled that PR before reading your comment

**Comment by eliezio** [comment]
PR superseded by #1885

**Pull request closed without merging**